migrations/versions/2019_04_01_fkey_cascades_cascade_foreign_key_relationships_for_.py

```diff
@@ -0,0 +1,217 @@
+"""Cascade foreign key relationships for deleting measure versions
```

Comment:
I really hope Alembic did most of the work for you for this migration :see_no_evil: 

It looks fine at a glance, but if it's hand-crafted let me know and I'll give it a more thorough look over.

Reply:
This was 100% alembic.